tests/unit/parser.js

```diff
@@ -5271,50 +5271,9 @@ exports.classes = function (test) {
 };
 
 exports["class and method naming"] = function (test) {
-  var code = [
```

Comment:
I'd personally prefer that this was not moved into a fixture. It's easier to read in one file.

I don't see any reason to add the extra stuff to this same code though, you can write a new test function for that